0_portfolio_input_check_functions.R

```diff
@@ -64,13 +64,39 @@ clean_portfolio_col_types <- function(portfolio, grouping_variables){
   
   portfolio$currency <- if_else(portfolio$currency == "Euro","EUR",portfolio$currency)
   
+  if(is.character(portfolio$investor_name) == FALSE) {
```

Comment:
What will happen if one of these columns is not in the file? Or is this check previously?

Reply:
That is being checked in the function before this one (check_missing_cols() I believe). So theoretically it should not happen